.github/workflows/scheduled.yml

```diff
@@ -124,10 +124,10 @@ jobs:
           brew install curl
           echo "HOMEBREW_FORCE_BREWED_CURL=1" >>"${GITHUB_ENV}"
 
-      - name: Install and use Homebrew svn if needed
+      - name: Install Homebrew subversion if needed for downloading sources
         run: |
-          if brew ruby -e 'exit 1 if Formula[ARGV.first].deps.none? { |d| d.name == "subversion" && d.implicit? }' "$FORMULA"; then
-            brew install svn
+          if brew deps --include-implicit --direct "$FORMULA" | grep -Fqx subversion; then
```

Comment:
It seems like we should be comparing the output of `brew deps` with and without `--include-implicit` to check if `subversion` is an implicit dependency, no?

Reply:
How about something like
```
comm -23 <(brew deps --include-implicit -direct "$FORMULA" | sort) <(brew deps --direct "$FORMULA" | sort)
```